application/frontend/config/main.php

```diff
@@ -47,20 +38,18 @@
             'enableSession' => false,
             'loginUrl' => null,
         ],
-        'sessionCache' => [
-            'class' => 'yii\caching\MemCache',
-            'servers' => $memcacheConfig,
-        ],
         'session' => [
-            'class' => 'yii\web\CacheSession',
-            'cache' => 'sessionCache',
-            'cookieParams' => [// http://us2.php.net/manual/en/function.session-set-cookie-params.php
-                'lifetime' => $sessionLifetime,
-                'path' => '/',
-                'httponly' => true,
-                'secure' => $frontCookieSecure,
-            ],
-
+            'class' => 'yii\web\DbSession',
+             'db' => [
+                 'class' => 'yii\db\Connection',
+                 'dsn' => sprintf('mysql:host=%s;dbname=%s', $mysqlHost, $mysqlDatabase),
+                 'username' => $mysqlUser,
+                 'password' => $mysqlPassword,
+                 'charset' => 'utf8',
+                 'emulatePrepare' => false,
+                 'tablePrefix' => '',
+             ],
```

Comment:
Looks like these ended up with an extra space of indentation:
```suggestion
            'db' => [
                'class' => 'yii\db\Connection',
                'dsn' => sprintf('mysql:host=%s;dbname=%s', $mysqlHost, $mysqlDatabase),
                'username' => $mysqlUser,
                'password' => $mysqlPassword,
                'charset' => 'utf8',
                'emulatePrepare' => false,
                'tablePrefix' => '',
            ],
```